src/app/core/entity-list/entity-list/entity-list.component.ts

```diff
@@ -387,4 +398,14 @@ export class EntityListComponent<T extends Entity>
   onRowClick(row: T) {
     this.elementClick.emit(row);
   }
+
+  private dynamicComponents: [string, AsyncComponent][] = [
```

Comment:
Let's create a new folder and module under `src/app/features/de-duplication/` for this feature (including services etc.) and also do the component registration in the module. This is the pattern we have started establishing for features that can be separated from core platform parts